app/src/main/res/layout/item_comment.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/bg_comment_item">
+
+    <android.support.v7.widget.AppCompatImageView
+        android:id="@+id/commentUserIcon"
+        android:layout_width="@dimen/comment_user_image_size"
+        android:layout_height="@dimen/comment_user_image_size"
+        android:layout_margin="@dimen/comment_item_margin"
+        android:contentDescription="@null"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <android.support.v7.widget.AppCompatImageView
+        android:id="@+id/commentMenu"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:background="?selectableItemBackgroundBorderless"
+        app:srcCompat="@drawable/ic_more_without_padding"
+
+        android:paddingLeft="10dp"
+        android:paddingRight="10dp"
+        android:paddingTop="3dp"
```

Comment:
Perhaps we could take out these values into dimens?

Reply:
It's icon related parameters and they mean nothing